src/model/OrganizationSubscription.js

```diff
@@ -1,4 +1,5 @@
 import isUrl from "is-url";
+import _urlParser from "../urlParser";
```

Comment:
I think it's not used

Reply:
There were several unused imports, so I just went ahead and removed them all